packages/components/src/internal/components/assay/SpecialtyAssayPanel.tsx

```diff
@@ -31,27 +30,39 @@ export const SpecialtyAssayPanel: FC<SpecialtyAssayPanelProps> = memo(props => {
         [onChange]
     );
 
+    const noPremiumClass = useMemo(() => {
```

Comment:
I was confused by this const name since there wasn't any use of the `premium` prop in the body.

Reply:
I changed it to premiumInfoClass.  Hopefully that is clearer.